vet.sh

```diff
@@ -1,71 +1,75 @@
 #!/bin/bash
 
-set -ex  # Exit on error; debugging enabled.
-set -o pipefail  # Fail a pipe if any sub-command fails.
+set -ex         # Exit on error; debugging enabled.
```

Comment:
Sorry, but can you revert all the lines that didn't need to be changed? It will kill `blame`.